### PR TITLE
Make `rake build` work with rake 11+ 3683

### PR DESCRIPTION
**Pull request by kirushik**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@kalabiyau will review your pull request :dancers: check https://trello.com/c/DGyJX42F/3683-fix-jenkins-connect-builds
  @kalabiyau: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

File: Rakefile [hunk 1]
```diff
@@ -3,6 +3,13 @@ require 'suse/connect'
 require 'rspec/core/rake_task'
 require 'date'
 
+module MonkeypatchRakeLastComment
```
Comment (kalabiyau) [inline Rakefile]: I would miss a hint on the reasons why that monkey patching is needed. Could you provide one?

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
http://river.suse.de/job/SCC-Connect-Prophet/191413/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests.
( Failure: Merged dd0631895b45d6598a5f7cad428ea305d7c8c9a5 into 01a296b795679c342ca2b1a294c87083a12b9f52 )

File: Rakefile (hunk 1 above)
Comment (kirushik) [inline Rakefile]: This is the error in Prophet: http://river.suse.de/job/SCC-Connect/144/console (`method missing` at the very bottom).

The `last_comment` method has been silently removed from Rake 11.0.1, then restored with a deprecation warning: https://github.com/ruby/rake/blob/v11.1.2/lib/rake/task_manager.rb#L9-L12

We currently happen to use the affected 11.0.x versions in our build pipeline, so this is really needed.
I'll add that as a code comment.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/SCC-Connect-Prophet/191448/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 837b44c2f6f205a520aa195ee49fdfb12127dc38 into 01a296b795679c342ca2b1a294c87083a12b9f52 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/SCC-Connect-Prophet/191453/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 98166736ae8a3c5a4c50fbc0361fd9fac4d814db into 01a296b795679c342ca2b1a294c87083a12b9f52 )

**Pull request merged**